docker-compose.ssl.yml

```diff
@@ -3,7 +3,7 @@ services:
   worker: &worker
     environment:
       - ACCOUNT_DEFAULT_HTTP_PROTOCOL=https
-      - ALLOWED_HOSTS=developer.127.0.0.1.nip.io,demos.developer.127.0.0.1.nip.io,api,localhost,web
+      - ALLOWED_HOSTS=developer.127.0.0.1.nip.io,beta.developer.127.0.0.1.nip.io,wiki.developer.127.0.0.1.nip.io,demos.developer.127.0.0.1.nip.io,api,localhost,web
```

Comment:
I was going to suggest overrides for ``WIKI_HOST``, ``BETA_HOST``, etc., but I suspect you are relying on the default behavior of prepending ``wiki``, ``beta`` to ``DOMAIN``?

Reply:
@jwhitlock Yes, exactly.